kahuna/public/js/components/gr-add-label/gr-add-label.js

```diff
@@ -5,25 +5,40 @@ import '../../services/label';
 import './gr-add-label.css!';
 import template from './gr-add-label.html!text';
 
+import '../../directives/gr-auto-focus';
+
 export var addLabel = angular.module('gr.addLabel', [
-    'kahuna.services.label'
+    'kahuna.services.label',
+    'gr.autoFocus'
 ]);
 
 addLabel.controller('GrAddLabelCtrl', ['$window', 'labelService',
     function ($window, labelService) {
+
+
         function saveFailed() {
             $window.alert('Something went wrong when saving, please try again!');
         }
 
         let ctrl = this;
 
-        ctrl.addLabel = () => {
-            let label = ($window.prompt('Enter a label:') || '').trim();
-            if (label) {
-                ctrl.addLabels([label]);
+        ctrl.active = false;
+
+        ctrl.addLabel = function (){
+            let labelList = ctrl.newLabel.split(",").map(e => e.trim());
+
+            if(labelList) {
+                ctrl.addLabels(labelList);
+                ctrl.newLabel="";
+                ctrl.active = false;
             }
         };
 
+        ctrl.cancel = function () {
+            ctrl.newLabel="";
```

Comment:
Does our jscs enforce spaces around `=`? Would be nice if it did for consistency, ease of searching, etc.